README.md

```diff
@@ -525,16 +559,16 @@ See the example [custom node](#nodes).
 
 ### Page table of contents
 
-Each proccessed page automatically exports a `headings` property with all headings on the page and IDs for each.
-Add IDs with [annotations](https://markdoc.dev/docs/syntax#annotations) or they are generated automatically.
+When you have the [`headingIds` option](#heading-ids) set to `true`,
```

Comment:
```suggestion
When you have the [`headingIds` option](#heading-ids) enabled`,
```

Does the table of contents example work with both `headingIds: true` and `headingIds: customSluggerFunction`? If so, I think it's better with `enabled` than `set to true`.